openedx/features/course_experience/url_helpers.py

```diff
@@ -32,70 +48,27 @@ def get_courseware_url(
     course that the block is in, the requesting user, and the state of
     the 'courseware' waffle flags.
 
-    If you know that you want a Learning MFE URL, regardless of configuration,
-    then it is more performant to call `get_learning_mfe_courseware_url` directly.
+    If you know the specific Sequence or Sequence/Unit you need to redirect to,
+    and you know that you want a Learning MFE URL regardless of configuration,
+    then it is more performant to call `make_learning_mfe_courseware_url` directly.
```

Comment:
Nit: `If redirecting to a specific Sequence or Sequence/Unit in a Learning MFE regardless of configuration, call make_learning_mfe_courseware_url directly for better performance.`